#/** ------ 停车位信息 ------ */
geometry  sGeo
coord     sPos
float32   u4RoadS
float32   u4RoadT
uint64    u8SlotId
